packages/utils/calculateCartTotal.js

```diff
@@ -68,7 +79,6 @@ const calculateCartTotal = (cart) => {
 
   const donation = get(cart, 'donation', 0)
 
-  const totalTaxes = Math.ceil(taxRate * subTotal)
```

Comment:
Why did this line get removed but left in the return statement on line 93 below?
Have you checked there is no caller of this method relying on the totalTaxes field?

Reply:
The 'totalTaxes' field is still calculated -- see line 34 in the modified file. I made 2 changes in relation to it:
1) Moved it 'upwards' (to line 34) to aid in the calculation of the `prediscountTotal`.
2) Adjusted the math because `taxRate` is a "fixed point" number as mentioned in lines 29-32 of the modified file.

Reply:
Ah... yep. My bad!